EthOn.rdf

```diff
@@ -1935,9 +1935,10 @@
 
     <owl:Class rdf:about="http://ethon.consensys.net/ExternalAccount">
         <rdfs:subClassOf rdf:resource="http://ethon.consensys.net/Account"/>
+        <simpleDefinition>An Ethereum Account usually controlled by a person. It is identified by an address (0x...) and a pubblic/private key-pair.</simpleDefinition>
```

Comment:
A few minor edits I would make here: 
- 'Account' should be lower case as we discussed earlier
- Typo, should be 'public/private keypair'